.github/workflows/welcome-external-pr.yml

```
@@ -0,0 +1,53 @@
+name: External contribution guidelines
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+
+env:
+  ORGANIZATION: Ethereum
+  DRY_RUN: true
+
+jobs:
+  comment-external-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get organization members
+        id: get_members
+        env:
+          GH_TOKEN: ${{ secrets.READ_ORG }}
+          CONTRIBUTOR: ${{ github.event.pull_request.user.login }}
+        run: |
+          gh api graphql \
+            --raw-field organization="$ORGANIZATION" \
+            --raw-field query='
+              query($organization: String!, $cursor: String) {
+                organization(login: $organization) {
+                  membersWithRole(first: 100, after: $cursor) {
+                    pageInfo {
+                      hasNextPage,
+                      endCursor
+                    }
+                    nodes {
+                      login
+                    }
+                  }
+                }
+              }' > org_members.json
+          echo "CONTRIBUTOR_IS_ORG_MEMBER=$(jq --arg contributor $CONTRIBUTOR '.data.organization.membersWithRole | any(.nodes[].login; . == $contributor)' org_members.json)" >> $GITHUB_OUTPUT
+
+      - name: Comment on external contribution PR
+        if: ${{ steps.get_members.outputs.CONTRIBUTOR_IS_ORG_MEMBER == 'false' }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR: ${{ github.event.pull_request.html_url }}
+        run: |
+          echo "Commenting new external PR: $PR"
+          if [[ $DRY_RUN == 'false' ]]; then
+            gh pr edit $PR --add-label "external contribution :star:"
+            gh pr comment $PR --body "Hi, thank you for your contribution. A Solidity team member will shortly look at it.
+            Please, make sure you have read and followed our [contributing guidelines](https://docs.soliditylang.org/en/latest/contributing.html) and also our [review checklist](https://github.com/ethereum/solidity/blob/develop/ReviewChecklist.md) so the process of accepting your contribution can be speedup.
+            If you have any questions, you can reach out to the core team using the [solidity-dev Matrix channel](https://matrix.to/#/#ethereum_solidity-dev:gitter.im)"
```

This will not render that well on github (e.g. the indent will be part of the comment, and comments have hard line breaks) and it's not great to read in the code either. How refactoring it this way?

```suggestion
            comment_body=(
                "Hi, thank you for your contribution. A Solidity team member will shortly look at it."
                "\n"
                "Please, make sure you have read and followed our "
                "[contributing guidelines](https://docs.soliditylang.org/en/latest/contributing.html) "
                "and also our [review checklist](https://github.com/ethereum/solidity/blob/develop/ReviewChecklist.md) "
                "so the process of accepting your contribution can be speedup."
                "\n"
                "If you have any questions, you can reach out to the core team using the "
                "[solidity-dev Matrix channel](https://matrix.to/#/#ethereum_solidity-dev:gitter.im)"
            )
            gh pr comment $PR --body "$(IFS='' ; echo -e "${comment_body[*]}")"
```

---

Indeed. I applied your suggestions :)